### ISSUE TITLE
Add IP Intelligence policy support to Virtual Server declaration

### ISSUE THREAD

**Issue by aknot242** [F5Networks/f5-appsvcs-extension#137]
##### Do you already have an issue opened with F5 support?
Github Issues are consistently monitored by F5 staff, but should be considered as best effort only and you should not expect to receive the same level of response as provided by F5 Support. Please [open an case](https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

##### ISSUE TYPE

 - Feature Idea

##### AS3 BUILD/ VERSION

```
N/A
```

##### BIGIP VERSION

```
N/A
```

##### SUMMARY

```
Please add IP Intelligence policy support to the appropriate AS3 virtual server (service) classes.
```

##### STEPS TO REPRODUCE

```

```

##### EXPECTED RESULTS

```

```

##### ACTUAL RESULTS

```

```

**Comment by dstokesf5** [comment]
Do you need AS3 to create the IP Intelligence policy, or is just referencing a pre-existing policy sufficient for your use case?

**Comment by aknot242** [comment]
Both, but delivering the 'reference a pre-existing policy' first is certainly welcome progress. Thanks!

**Comment by dstokesf5** [comment]
It looks like this is already in our internal product backlog:
Referencing existing policies: AUTOTOOL-602
Creating new policies: AUTOTOOL-49

**Comment by jmcalalang** [comment]
Second the request fo this!

**Comment by dstokesf5** [comment]
This has been resolved in the 3.35.0 release.
